templates/download/server/manual.html

```diff
@@ -1,204 +1,405 @@
-      <p>
-        <a href="{{ releases.latest.release_notes_url }}">
-          Read the Ubuntu Server {{ releases.latest.short_version }} release notes&nbsp;&rsaquo;
-        </a>
-      </p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Ubuntu {{ releases.latest.full_version }}</h2>
+      </div>
+      <div class="col">
+        <p>The latest version of Ubuntu Server, including nine months of security and maintenance updates, until July 2024.</p>
+        <hr />
+        <form action="/download/server" method="post" style="display: inline">
+          <input type="hidden" name="version" value="{{ releases.latest.full_version }}">
+          <input type="hidden" name="architecture" value="amd64">
+          <input type="hidden" name="next-step" value="download">
+          <button style="margin-right: 1rem;" type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });">Download {{ releases.latest.full_version }}</button>
+        </form>
+        <span class="u-text--muted">ISO image, {{ releases.lts.iso_download_size }}</span>
+        <p><a href="https://discourse.ubuntu.com/t/mantic-minotaur-release-notes/35534">Read the 23.10 release notes&nbsp;&rsaquo;</a></p>
+      </div>
     </div>
-    <div class="col-4 col-start-large-8 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/57a36cad-Minotaur_circular.png",
-        alt="",
-        width="260",
-        height="260",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-  {% endif %}
-</section>
-
-<section class="p-strip--light u-no-padding--bottom" id="downloads">
-  <div class="row">
-    <div class="col-10">
-      <h2>Alternative downloads</h2>
+  </section>
+  
+  <section class="p-section">
+    <div class="row--25-75">
+      <hr class="p-rule" />
+      <div class="col">
+        <h3 class="p-text--small-caps">Alternative downloads</h3>
+      </div>
+      <div class="col">
+        <div class="row p-section--shallow">
+          <div class="col-3">
+            <h4 class="p-heading--5">BitTorrent</h4>
+          </div>
+          <div class="col-6">
+            <p>BitTorrent sometimes enables higher download speeds and more reliable downloads of large files.</p>
+            <hr />
+            <a class="p-button" href="https://releases.ubuntu.com/{{ releases.lts.short_version }}/ubuntu-{{ releases.lts.full_version }}-live-server-amd64.iso.torrent">Torrent for {{ releases.lts.full_version }} LTS</a>
```

Comment:
Same here, this appears as 22.04.4

Reply:
Here copy doc was wrong: torrent is for the point release, so MariaPaula's approach is correct. I updated the copy doc.